### ISSUE TITLE
Beginner's Area

### ISSUE THREAD

**Issue by theianrey** [HeapsIO/hide#70]
Hi!
I'm quite new to Heaps engine as well as in Haxe. I've tried the tutorials on how to start with the game engine. Then I came across on the HIDE application and I'm lost. Is there any tutorial on how to use HIDE application in creating levels or even the basics workflow of it?

As of now, all I can see on the IDE are the grids and a box polygon. even though I'm adding some mesh or terrain, even a camera. I don't see them in the editor. 😞 

I hope you can enlighten me on this matter.
Thanks

**Comment by AnxiousDarkly** [comment]
This probably is not the answer you want but this stack (haxe, heaps, hide) has almost zero documentation, imo it isn't friendly to new users in any way, so unless you are comfortable diving into the codebases, with little to no direction, you will likely be better off going with a more mainstream game engine.

**Comment by joereynolds** [comment]
Piggybacking off of the above, I've found HaxeFlixel to be a well documented beginner friendly framework.

I tried to get into Heaps myself and even wrote some [simple samples for beginners](https://github.com/joereynolds/haxe-heaps-examples) to try and help in some way but the lack of docs was something I fought against the entire way. In the end there were simple things that I wanted to use (a camera) that were built into Heaps but had no documentation. I raised this and was told to read the code to understand it. They could gain many more users if they polished off their API documentation and a basic game from start to finish IMO. It's a shame because Heaps looks promising.
